scripts/requious/replicator_stat.zs

```diff
@@ -5,5 +5,5 @@ mods.zenutils.PlayerStat.create(
   'stat.replications',
   crafttweaker.text.ITextComponent.fromTranslation('stat.replications'),
   function(mb as int) as string {
-    return mods.zenutils.StaticString.format('%,.2f', 0.001 * mb);
+    return mods.zenutils.StaticString.format('%,.2f', [0.001 * mb]);
```

Comment:
Blame my zenscript internal changes. I have to wrap it as an array, vararg don't work for this function now. 😢

Reply:
Does this means i need to change `StaticString.format` arguments in my code everywhere i'm using them?
And, is this vararg thing must be addressed to any other functions?

Reply:
zs only reported one error of this file, others are fine. So you only need to change it. I assume it is because we pass double to the function, zs needs cast it to Object first. If we use string as parameter, it is fine.